drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelViewImpl.html

```diff
@@ -100,8 +100,31 @@
         <!-- content for cheatsheet tab -->
         <div id="kieScenarioCheatsheetTabContent"
              class="tab-pane fade"
-             aria-labelledby="Scenario Cheatsheet">
-            <p>Create a row for each scenario, and input values using the following syntax.</p>
+             aria-labelledby="Scenario Cheatsheet" style="padding: 1em">
+            <p>Use contextual menu to add/remove columns and rows</p>
+
+            <p>Left click on the header of a column and then edit the mapping with a field through the right panel</p>
+
+            <p>Double click on the field to set the mapping</p>
+
+            <p>Each cell can contain an expression using the following syntax</p>
+
+            <ul>
+                <li><i>= value (or no symbol)</i>: to specify equals to a value. This is the only operator supported under GIVEN section and it is the default</li>
+                <li><i>! value</i>: to specify not equals to a value. It can be composed with other operator</li>
+                <li><i>< / > / <= / >= value</i>: to specify a comparison</li>
+                <li><i>[ value1, value2, value3 ]</i>: to a list of values. The semantic is "one of them"</li>
+                <li><i>expression1 ; expression2 ; expression3</i>: to specify a list of expressions. The semantic is "all of them"</li>
```

Comment:
_all of them_ -> _all of them must hold_

Reply:
Done

Reply:
Change line 117 to >> specifies a list of expressions.

Also, from what I understand, this operator is working as the _logical AND_ operator. So instead of saying "all of them must hold", we can simply say >>> "This operator works like the logical AND operator." or something similar.
Similarly, we can say in the previous line that its like the logical OR. 

Correct me if I am wrong on this.